doc/ref/configuration/master.rst

```diff
@@ -1925,6 +1925,42 @@ the cloud profile or master config file, no templating will be performed.
 
     userdata_template: jinja
 
+.. conf_master:: jinja_line_statement_prefix
+
+``jinja_line_statement_prefix``
+---------------------
+
+.. versionadded:: Oxygen
+
+Default: ``''``
+
+If this is set to a string that is not empty, any line starting with this prefix
+will be interpreted as a line statement by the jinja renderer.
+Defaults to ``''`` and corresponds
+to the Jinja environment init variable ``line_statement_prefix``.
+
+.. code-block:: yaml
+
+    jinja_line_statement_prefix: ''
```

Comment:
I would like to see here example from my PR.
(it took me some time to find what really works - not not everything works)